tools/flang2/flang2exe/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ set(SOURCES
   dtypeutl.cpp
   fenddf.cpp
   ilmutil.cpp
-  listing.cpp
+  listing.c
```

Comment:
This rename doesn't seem necessary at this point. In a future PR, perhaps we could move `listing.c` and `listing.cpp` from under `tools/flang1/flang1exe/` and `tools/flang2/flang2exe/`, and merge them into one file under `tools/shared/`.

Reply:
I think I did this to come up with a proper name mangling and resolve link errors.